gourmet/plugin_gui.py

```diff
@@ -61,6 +61,16 @@ def make_list_store (self, plugin_list):
                 )
         return ls
 
+    @staticmethod
+    def data_fun(col, renderer, mod, itr, _):
```

Comment:
if it's not too much of a hassle, could you annotate this with types? The ones you don't know or are not sure about you can annotate with `Any` and we can later make that an error in `mypy`.

Reply:
But I am not sure about the extraction. Do we plan on using this function anywhere else?

Reply:
The function is just a bit oversized to be a lambda, and I extracted it for sake of potential future enhancements. (The resulting output doesn't look fancy now, IMO.)

I annotated it, please have a look.